packages/node_modules/@webex/common/src/patterns.js

```diff
@@ -15,7 +15,7 @@ export default {
    * @todo add a better email address matcher
    * @type {RegExp}
    */
-  email: /^[^\s]+?@[^\s]+?$/,
+  email: /[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*/,
```

Comment:
Where did you grab the regex for this?

I usually check here first
https://ihateregex.io/expr/email-2

Reply:
I updated the regex based on the provided email, I also separated this for legacy support on regex requiring the older version.